code/modules/security_levels/security_level_datums.dm

```diff
@@ -0,0 +1,92 @@
+/**
+ * Security levels
+ *
+ * These are used by the security level subsystem. Each one of these represents a security level that a player can set.
+ *
+ * Base type is abstract
+ */
+
+/datum/security_level
```

Comment:
Missing epsilon and gamma levels (idc if they work without them, find out why it works without them, because it shouldnt)

Reply:
ugh fine

Reply:
Readded them and also i found out why it works without them (It didnt lol!)